smile_id_core/Utilities.py

```diff
@@ -244,18 +245,13 @@ def validate_id_params(
             and not id_info_params.get("entered")
         ):
             return
-
-        required_fields = ["country", "id_type"]
-        if job_type != JobType.DOCUMENT_VERIFICATION:
-            required_fields += ["id_number"]
-
-        for field in required_fields:
-            if field in id_info_params:
-                if id_info_params[field]:
-                    continue
-                raise ValueError(f"key {field} cannot be empty")
-            else:
-                raise ValueError(f"key {field} cannot be empty")
+        
+        if job_type != JobType.DOCUMENT_VERIFICATION and not id_info_params.get("id_number"):
```

Comment:
Can you invert the logical condition and check for if job type == document verification? Everything except checks often cause problems later.